Coralogix.podspec

```
@@ -32,10 +32,29 @@ Pod::Spec.new do |spec|
   spec.ios.deployment_target = "13.0"
 
-  spec.source_files  = 'Coralogix/Sources/**/*.{swift,h}'
+  spec.source_files  = 'Coralogix/Sources/**/*.{swift}'
   spec.exclude_files = 'Coralogix/Sources/Exclude'
 
   spec.static_framework = true
   spec.dependency 'PLCrashReporter', '~> 1.11.1'
+
+  spec.source_files = [
+    'Coralogix/Sources/**/*.swift',
+    'CoralogixInternal/Sources/**/*.swift'
```

RuboCop (1.73)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)